microcosm_metrics/decorators.py

```diff
@@ -26,13 +27,15 @@ def wrapper(*args, **kwargs):
                 try:
                     return classifier(*args, **kwargs)
                 finally:
+                    environment = environ.get("MICROCOSM_ENVIRONMENT", "missing")
```

Comment:
Do we want to use `missing` instead of doing a noop and/or failing?

Reply:
We liked `missing`, or some other explicit failure indication, because it will show up on the board, and we'll know that something is wrong.

Reply:
Let's go with `undefined` then.